### PR TITLE
rpg-skilltrees: design and implement example file(s) for complex skills

### LINKED ISSUE #1465: rpg-skilltrees: design and implement example file(s) for complex skills
We will write example files for the complex skills features implemented in #1382. We aim to have at least one example for each type of complex skill: sequential, combined, conditional, random. We will write our examples in accordance to the examples that will be introduced in the upcoming wiki page for complex skills (see #1461).

Team members: Arya, Chelsea, Kabir

### PR DESCRIPTION
**Pull request by KabirDubey**
The progress made in this PR corresponds to issue #1465. We wrote example files for the complex skills features implemented in https://github.com/uchicago-cs/chiventure/pull/1382. We wrote examples for sequential, combined, and conditional complex skills. See https://github.com/uchicago-cs/chiventure/issues/1461 for wiki documentation. These examples will be used in the 2022 demo.

### CODE REVIEW

File: CMakeFiles/TargetDirectories.txt [hunk 1]
```diff
@@ -0,0 +1,186 @@
+/home/kabirdubey/cs220/chiventure/CMakeFiles/rebuild_cache.dir
```
Comment (nbilaver) [inline CMakeFiles/TargetDirectories.txt]: Hey, I think you didn't mean to add all of the compiled files, especially files like this that have your directory information. Pretty sure you did git add -A after you compiled. Delete all of these files that aren't a part of the code, like the makefiles, the autogenerated cmake files, and the like.

**Review by nbilaver — APPROVED** [review]
Everything looks pretty good! Don't see anything wrong here.

File: CMakeFiles/TargetDirectories.txt (hunk 1 above)
Comment (KabirDubey) [reply CMakeFiles/TargetDirectories.txt]: Thanks, fixed.

**Comment by KabirDubey** [comment]
Finalized the changes, but didn't make a new example for simplicities sake. Now awaiting @jacknugent1529 review and approval.

**Review by jacknugent1529 — APPROVED** [review]
Nice work on this! It looks good to me

**Comment by KabirDubey** [comment]
Closing statement: Arya wrote examples for combined and sequential skills. Kabir wrote the conditional skill example and re-organized the module. Chelsea ensured the new files were integrated correctly and finalized the additions.

**Comment by jacknugent1529** [comment]
Issue Score: Excellent

Comments: Nice work creating examples! I think this will be especially useful for teams in the future wanting to use complex skills.

**Pull request merged**